draft-ietf-ppm-dap.md

```diff
@@ -1667,9 +1673,10 @@ enum {
   hpke_unknown_config_id(4),
   hpke_decrypt_error(5),
   vdaf_prep_error(6),
-  task_expired(7),
+  task_ended(7),
```

Comment:
nit: I think we should just keep this as `task_expired`. Its meaning is clear enough, and there would be one less thing that implementations need to change.